### PR TITLE
fix(DTFS2-5963): Amendments details reverting when multiple single amendment changes

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue:
* When having multiple single change amendments, if latest amendment was only value, coverEndDate would revert to facility submission value

# Changes made:
* 2 new endpoints for getting latest amendment value and coverEndDate
* Modified mapping to take new object instead
* Updated api-tests
* Added e2e test

### CODE REVIEW

File: dtfs-central-api/src/v1/routes/tfm-routes.js [hunk 1]
```diff
@@ -413,7 +413,8 @@ tfmRouter.route('/amendments/status/in-progress').get(tfmGetAmendmentController.
 tfmRouter.route('/facilities/:facilityId/amendment').get(tfmGetAmendmentController.getAllAmendmentsByFacilityId);
 tfmRouter.route('/facilities/:facilityId/amendment/status/in-progress').get(tfmGetAmendmentController.getAmendmentInProgress);
 tfmRouter.route('/facilities/:facilityId/amendment/status/completed').get(tfmGetAmendmentController.getAllCompletedAmendmentsByFacilityId);
-tfmRouter.route('/facilities/:facilityId/amendment/status/completed/latest').get(tfmGetAmendmentController.getLatestCompletedAmendment);
+tfmRouter.route('/facilities/:facilityId/amendment/status/completed/latest-value').get(tfmGetAmendmentController.getLatestCompletedValueAmendment);
+tfmRouter.route('/facilities/:facilityId/amendment/status/completed/latest-date').get(tfmGetAmendmentController.getLatestCompletedDateAmendment);
```
Comment (IgnatG) [inline dtfs-central-api/src/v1/routes/tfm-routes.js:417]: can we use `/latest-cover-end-date`?

File: trade-finance-manager-ui/server/api.js [hunk 2]
```diff
@@ -337,17 +337,32 @@ const getCompletedAmendment = async (facilityId) => {
   }
 };
 
-const getLatestCompletedAmendment = async (facilityId) => {
+const getLatestCompletedValueAmendment = async (facilityId) => {
   try {
     const response = await axios({
       method: 'get',
-      url: `${tfmAPIurl}/v1/facility/${facilityId}/amendment/status/completed/latest`,
+      url: `${tfmAPIurl}/v1/facility/${facilityId}/amendment/status/completed/latest-value`,
```
Comment (IgnatG) [inline trade-finance-manager-ui/server/api.js:344]: `/latest-cover-end-date` - to keep it consistent

File: dtfs-central-api/src/v1/routes/tfm-routes.js (hunk 1 above)
Comment (Zainzzkk) [reply dtfs-central-api/src/v1/routes/tfm-routes.js:417]: done

**Review by IgnatG — APPROVED** [review]

**Comment by IgnatG** [comment]
LGTM

**Pull request merged**